Improve the proxy certificate doc (#95)
newrelic/newrelic-fluent-bit-output: README.md

    # REMOVE following option when using it on Windows (see above)
    caBundleFile /path/to/proxy-certificate-bundle.pem
```
#### Certificates
Certificates can be trick because depends on the proxy (`HTTP` vs `HTTPS`), Linux, Windows, Logging Endpoint,
Infra-Agent Endpoint, etc. We will talk about these configurations below.

##### Linux
For `HTTP` proxies we don't need to setup any certificate. Out of the box the plugin will load the system certificates
and we will be able to send logs into the logging endpoint.

> For `HTTPS` proxies you can specify the self-signed certificate (PEM file) using either the `caBundleFile` or `caBundleDir`

Reviewer: ```suggestion
For `HTTPS` proxies you can specify the proxy self-signed certificate (PEM file) using either the `caBundleFile` or `caBundleDir`
```

parameters.

##### Windows
Similar to Linux, for `HTTP` proxies we don't need to setup any certificate. Out of the box the plugin will load
the system certificates.

> For `HTTPS` it require attention:

Reviewer: ```suggestion
For `HTTPS`, **special attention is required**. There are two approaches to configure it: by importing them to the system certificate pool (recommended), or by using the `caBundleFile`/`caBundleDir` options.

###### Approach 1: import the proxy certificate to the system pool (**recommended**)
```

###### Approach 1: import the proxy certificate to the system pool (**recommended**)

> 

Reviewer: ```suggestion
```

> The recommended process to import the self-signed certificate (PEM file) using the MMC tool. You can refer to
> [this link](https://www.ssls.com/knowledgebase/how-to-import-intermediate-and-root-certificates-via-mmc/), but in Step 2
> ensure to import it in your "Trusted Root Certification Authorities" instead of importing it in the
> "Intermediate Certification Authorities".

Reviewer: ```suggestion
The recommended process to import the proxy self-signed certificate (PEM file) using the MMC tool. You can refer to
[this link](https://www.ssls.com/knowledgebase/how-to-import-intermediate-and-root-certificates-via-mmc/), but in Step 2
ensure to import it in your `Trusted Root Certification Authorities` instead of importing it in the
`Intermediate Certification Authorities`.
```

> 

Reviewer: ```suggestion
###### Approach 2: using `caBundleFile`/`caBundleDir`

```

> If you are using `caBundleFile` or `caBundleDir`, different from Linux, on Windows we are not able to load both (the system
> certificates and the `caBundleFile`/`caBundleDir`). So you need to **assure** that all certificates are configured. You would
> end up having something like this:

Reviewer: ```suggestion
On Windows (differently from Linux) we cannot load both the certificates from system certificate pool and the one(s) specified via `caBundleFile`/`caBundleDir`. So, if you are using `caBundleFile` or `caBundleDir`, you must **ensure that the following certificates are placed in the same PEM file (when using `caBundleFile`) or in the same directory (when using `caBundleDir`)**:
```

> ###### Approach 2: using `caBundleFile`/`caBundleDir`

Reviewer: ```suggestion

###### Approach 2: using `caBundleFile`/`caBundleDir`
```

> 

Reviewer: ```suggestion
```

- The Proxy certificate (because it's a `HTTPS` proxy)
- The Logging Endpoint certificate (eg. `https://log-api.newrelic.com/log/v1`)

> - The Infra-Agent Endpoint certificate (eg. `https://infra-api.newrelic.com`)

Reviewer: Please do not include the `Infra-Agent` in this readme, since this is the output plugin documentation.

> The certificates can be checked though the `openssl` command:

Reviewer: ```suggestion
The Logging Endpoint certificate can be checked using the `openssl` command:
```
